src/test/java/com/futurecollars/lesson11/zad6/WorkoutTest.java

```diff
@@ -0,0 +1,22 @@
+package com.futurecollars.lesson11.zad6;
+
+import com.futurcollars.lesson11.zad6.WorkoutService;
+import com.futurcollars.lesson11.zad6.WorkoutComment;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EnumSource;
+
+public class WorkoutTest {
+    @ParameterizedTest
+    @EnumSource(value = WorkoutComment.class, names = {"BARDZO_DOBRA"})
+    void shouldGetResultWorkout(WorkoutComment workoutComment) {
+        int workoutLength = 65;
+        int calorieBurned = 350;
+        int pulse = 178;
+        // given
+        WorkoutService workoutService = new WorkoutService();
+        // when
+        WorkoutComment result = workoutService.getResultWorkout(workoutLength, calorieBurned, pulse);
+        Assertions.assertEquals(workoutComment, result);
```

Comment:
Dużo bardziej pasowałoby tutaj użycie MethodSource i podanie także danych z treningu, czyli na przykład:
Argument.of(65, 350, 178, WorkoutComment.BARDZO_DOBRA),
Argument.of(30, 200, 100, WorkoutComment.NISKA),

W tym momencie to nie jest test parametryczny, bo testujesz dla jednego przypadku.